cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/PythonLanguageFrontend.kt

```diff
@@ -356,13 +356,17 @@ class PythonLanguageFrontend(language: Language<PythonLanguageFrontend>, ctx: Tr
  * Returns the version info from the [TranslationConfiguration] as [VersionInfo] or `null` if it was
  * not specified.
  */
-fun getVersionInfo(config: TranslationConfiguration): VersionInfo? {
-    return config.symbols["PYTHON_VERSION_MAJOR"]?.toLong()?.let { major ->
-        val minor = config.symbols["PYTHON_VERSION_MINOR"]?.toLong()
-        val micro = if (minor != null) config.symbols["PYTHON_VERSION_MICRO"]?.toLong() else null
-        VersionInfo(major, minor, micro)
+val TranslationConfiguration.versionInfo: VersionInfo?
```

Comment:
I'll this and the other changes related to `VersionInfo` to a separate PR since the change is no longer needed for the functionality of this PR and thus introduces unnecessary changes